patches/chrome-android-java-src-org-chromium-chrome-browser-notifications-channels-ChannelDefinitions.java.patch

```diff
@@ -1,32 +1,33 @@
 diff --git a/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelDefinitions.java b/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelDefinitions.java
-index 62d219a61441e8d26ef71f76e2df45f17de4a4e8..98f017a2a82e34794e1fc9fa4e6c569dbe821f62 100644
+index 62d219a61441e8d26ef71f76e2df45f17de4a4e8..7095fa29578868d6eebd43476403c97bf9329b85 100644
 --- a/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelDefinitions.java
 +++ b/chrome/android/java/src/org/chromium/chrome/browser/notifications/channels/ChannelDefinitions.java
-@@ -55,6 +55,7 @@ public class ChannelDefinitions {
+@@ -55,6 +55,8 @@ public class ChannelDefinitions {
       * more detailed instructions.
       */
      @StringDef({ChannelId.BROWSER, ChannelId.DOWNLOADS, ChannelId.INCOGNITO, ChannelId.MEDIA,
 +            BraveChannelDefinitions.ChannelId.BRAVE_ADS,
++            BraveChannelDefinitions.ChannelId.BRAVE_ADS_BACKGROUND,
```

Comment:
can you put these both on one line? Otherwise lgtm

Reply:
it will break linter's rule: no more than 80 symbols per line

Reply:
Patches should have as few changes as possible so the idea here is to reduce the lines from 2 to 1 (which will not fail Linter)

Reply:
per @serg: "ignore linter in patches". 
Fixed.